app/models/linkage_group.rb

```diff
@@ -0,0 +1,13 @@
+class LinkageGroup < ActiveRecord::Base
+
+  has_and_belongs_to_many :linkage_maps,
+                          join_table: 'map_linkage_group_lists',
+                          foreign_key: 'linkage_group_id'
+
+  has_many :map_positions
+
+  has_many :map_locus_hits, foreign_key: 'linkage_group_id'
```

Comment:
If I am right, you don't need to specify foreign_key: 'linkage_group_id' here or in the habtm above, either.